docs/installation.rst

```diff
@@ -18,7 +18,32 @@ environment variable to contact the docker daemon.
 -  **Windows**: `Docker
    For Windows (create an account & follow through to download from the Docker Store) <https://www.docker.com/docker-windows>`__
 -  **Linux**: Check your distro’s package manager (e.g. yum install docker)
+      *** for Centos 7.5 System requirements are::
+         yum install gcc zip py-pip py-setuptools ca-certificates groff  python-dev g++ make docker epel-release python-pip python-devel\
+                     python-tools
+          **run post install of above**
+          pip install --upgrade pip
+          pip install --upgrade setuptools
+          pip install --upgrade aws-sam-cli
+        *** if you want to use the lambda-local option(without running it as root) you will need to add your user to the docker group ***
+         usermod -a -G Docker yourUserName
+      ***Docker if you would like to run SAM-CLI in a docker container (lambda-local will probably act weird)***

```

Comment:
Can you remove the Dockerfile related docs here? While you can use parts of SAM CLI in docker, all features and commands will not work. Before we add running SAM CLI in docker, we need to ensure all commands work as expected.